cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java

```diff
@@ -279,14 +279,14 @@ public ServiceTicket grantServiceTicket(
         getAuthenticationSatisfiedByPolicy(ticketGrantingTicket, new ServiceContext(service, registeredService));
 
         final String uniqueTicketIdGenKey = service.getClass().getName();
-        if (!this.uniqueTicketIdGeneratorsForService.containsKey(uniqueTicketIdGenKey)) {
-            logger.warn("Cannot create service ticket because the key [{}] for service [{}] is not linked to a ticket id generator",

```

Comment:
Not sure I am with you on this one. This pull intends to provide a default generator in order to relieve someone from having to define one explicitly. That's not a warning to me because it's no longer the requirement (Specially if the to-be-configured generator is going to look exactly as the default anyway)